packages/manager/modules/bm-server-components/src/service-status/service-status.scss

```diff
@@ -0,0 +1,4 @@
+.override-flex {
+  display: inherit;
```

Comment:
**nitpick:** Instead of adding `inherit` is it possible to be specific on which display we expect here?